components/ui-fragments/Input.js

```diff
@@ -31,14 +32,25 @@ const SearchButton = styled(IconButton)`
   width: 40px;
 `
 
-export const SearchInput = ({ onChange, value, placeholder, onClick }) => (
-  <SearchInputWrapper>
-    <HalfInput
-      onChange={onChange}
-      value={value}
-      placeholder={placeholder}
-      title={placeholder}
-    />
-    <SearchButton onClick={onClick} icon={SearchIcon} aria-label="Search" />
-  </SearchInputWrapper>
-)
+export const SearchInput = ({ onChange, value, placeholder, onClick }) => {
+  const inputElement = useRef(null)
+
+  useEffect(() => {
+    if (inputElement.current) {
+      inputElement.current.focus()
+    }
+  }, [])
```

Comment:
Will this focus on the input on every render? Maybe this could be done only when 'value' has changed, cause then a user has either typed something or pressed a character button.